src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -149,15 +135,14 @@ export const updateTrackStatesAndSave: ActionCreator<
 export const toggleBrowserNav = createAction(
   'browser/toggle-browser-navigation',
   (resolve) => {
-    return () => resolve(undefined, getBrowserAnalyticsObject('Navigation'));
+    return () => resolve(undefined);
   }
 );
 
 export const updateBrowserNavStates = createAction(
   'browser/update-browser-nav-states',
   (resolve) => {
-    return (browserNavStates: BrowserNavStates) =>
-      resolve(browserNavStates, getBrowserAnalyticsObject('Navigation'));
+    return (browserNavStates: BrowserNavStates) => resolve(browserNavStates);
   }
 );
```

Comment:
Most of the actions here, which just take a value and resolve this value without doing anything else can be re-written with `createStandardAction`. I don't know whether we _should_ (for some actions, you may argue that you will soon re-write them again, addind analytics data), but we definitely could. For those actions that aren't expected to be linked to analytics reporting, it will be a good idea.

Reply:
Update it to use createStandardAction.